code/game/gamemodes/devil/devilinfo.dm

```diff
@@ -383,6 +383,8 @@ var/global/list/lawlorify = list (
 		to_chat(owner.current, "<span class='danger'>You seem to have resurrected without your hellish powers.</span>")
 
 /datum/devilinfo/proc/check_banishment(mob/living/body)
+	if(QDELETED(body))	//Allow ressurection if the body has been destroyed
+		return 0
```

Comment:
No, this will break the banish ritual where you are SUPPOSED to destroy the body, take a look at line 415

Reply:
Good catch, thanks.